server/routes/notification.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const router = express.Router();
+const { getAllNotifications,
+  getAllUnreadNotifications,
+  createNotification,
+  markAsRead } = require("../controllers/notification");
+
+router.route('/create:id').post(createNotification);
```

Comment:
`/create:id` did you mean `/create/:id`?

Reply:
yep sorry